docs/docs/rnk-rpc-methods.md

```diff
@@ -0,0 +1,780 @@
+---
+title: RNK RPC Methods
+sidebar_position: 7
+description: Explore Reactive Network's RPC methods used for communication with nodes and ReactVMs.
+slug: /rnk-rpc-methods
+hide_title: true
+---
+
+## Overview
+
+This page provides an overview of Reactive Network's (RNK) RPC methods, which enable interaction with nodes and ReactVMs. The methods include transaction retrieval, log access, callback information, and more. Below is a detailed description of each method along with its parameters, usage examples, and responses.
+
+## rnk_getTransactionByHash
+
+Returns the details of a transaction for the specified ReactVM ID and transaction hash.
+
+### Parameters
+
+1. **rvmId**: `DATA`, 20 Bytes — The ReactVM ID associated with the transaction.
+2. **txHash**: `DATA`, 32 Bytes — The hash of the transaction to retrieve.
+
+**Example Parameters:**
+
+```json
+[
+  "0xc1d48a9173212567bd358e40c50bfe131a9fabf1",
+  "0x160f41dc30205679437b161ee1d0789c9c1235874896d47dd16dc8528f43f4dd"
+]
+```
+
+### cURL
```

Comment:
probably should be `####` because otherwise it ends up in ToC

Reply:
I made it so on purpose for quick navigation

Reply:
it's enough to quick navigate by method name

Reply:
this is unusable imo 

<img width="576" alt="Screenshot 2024-08-29 at 14 19 13" src="https://github.com/user-attachments/assets/29b95f88-3a83-46fd-9df1-0809f311b1be">